app/views/idv/in_person/state_id/show.html.erb

```diff
@@ -36,7 +36,7 @@
   <%= link_to(
         MarketingSite.help_center_article_url(
           category: 'verify-your-identity',
-          article: 'accepted-state-issued-identification',
+          article: 'accepted-identification-documents',
```

Comment:
I want to point out that this change was made due to the integration tests I added for  state_id_controller.

I was getting an `ActionView::Template::Error:Unknown help center article category verify-your-identity/accepted-state-issued-identification`, so I updated it to the `accepted-identification-documents` article that was defined in the `MarketingSite` service.